kratos/tests/mortar_mapper_python_tests/test_double_curvature_integration_triangle_quadrilateral.mdpa

```diff
@@ -0,0 +1,7896 @@
+	3249	17.9325	47.9422	28.8899
+	3252	36.1664	46.8698	48.1106
+	3253	32.0661	52.3598	39.1419
+	3255	16.5139	46.8706	29.2131
+	3259	36.4674	45.8886	50
+	3261	19.4851	49.8066	29.883
+	3262	27.2381	53.3918	34.1064
+	3263	32.9773	51.6882	41.8022
+	3264	34.9182	49.3837	45.8807
+	3269	17.5583	48.2895	30.1905
+	3270	23.4622	52.6581	31.6899
+	3273	15.9669	47.0865	30.6293
+	3274	28.3199	53.636	36.4869
+	3275	35.4828	48.3575	48.3595
+	3278	20.7822	51.4845	31.5465
+	3280	35.9571	47.3455	50
+	3282	24.4093	53.4156	34.0006
+	3283	16.8117	48.3868	31.7228
+	3285	29.3663	53.6644	39.0184
+	3288	34.8266	49.4465	48.3142
+	3289	15.4415	47.2939	32.0549
+	3292	18.3324	50.2642	32.7301
+	3293	30.5496	53.3348	41.6182
+	3294	35.2662	48.7258	50
+	3295	33.0947	51.5431	45.737
+	3299	25.3949	53.9163	36.4064
+	3300	16.4738	48.7159	33.0413
+	3302	21.5676	52.7627	33.929
+	3305	14.9426	47.4908	33.4915
+	3307	33.7762	50.8094	48.0907
+	3310	26.5735	54.2215	38.8963
+	3311	34.4152	50.0131	50
+	3314	15.7872	48.7801	34.5988
+	3316	22.4886	53.6736	36.2648
+	3318	14.4749	47.6754	34.9401
+	3320	32.6905	51.9043	48.0891
+	3321	27.7997	54.3095	41.5317
+	3323	17.282	50.6479	35.627
+	3325	30.5271	53.4517	45.2907
+	3326	19.759	52.6379	36.1906
+	3327	33.4193	51.1914	50
+	3329	15.5027	49.082	35.9399
+	3332	23.5682	54.2877	38.7794
+	3336	14.0426	47.846	36.4014
+	3338	31.3799	52.9432	48.3014
+	3340	14.8136	48.9791	37.4061
+	3341	24.8716	54.6403	41.3748
+	3342	32.294	52.2463	50
+	3346	20.7621	53.7552	38.67
+	3348	13.6495	48.0011	37.8754
+	3349	16.0057	50.3798	37.972
+	3352	30.1765	53.6823	48.0889
+	3353	18.1474	52.3816	38.5886
+	3356	14.6785	49.3731	38.8906
+	3357	31.0556	53.165	50
+	3360	26.8895	54.7447	45.3699
+	3361	13.2986	48.1396	39.3617
+	3362	21.9229	54.4892	41.2797
+	3364	28.8363	54.2901	47.7987
+	3369	14.1661	49.353	40.5033
+	3370	29.7154	53.9263	50
+	3371	19.126	53.5877	41.1406
+	3374	12.9925	48.2604	40.8593
+	3377	27.2963	54.7499	48.0795
+	3378	15.624	51.1299	41.5816
+	3379	14.0262	49.5797	41.8893
+	3382	28.2852	54.4991	50
+	3385	23.4006	55.0147	45.6841
+	3386	12.733	48.3629	42.3671
+	3388	25.6126	55.0128	47.8869
+	3392	13.5711	49.4629	43.5147
+	3393	26.7901	54.8703	50
+	3395	12.5211	48.4465	43.8834
+	3398	24.1833	55.1113	48.3926
+	3399	20.1894	54.577	45.474
+	3400	14.8479	50.9905	44.5784
+	3403	25.2626	55.0697	50
+	3404	13.5611	49.698	44.9258
+	3406	22.8749	55.1025	48.3124
+	3409	17.4271	53.2861	45.518
+	3410	12.3572	48.5112	45.4067
+	3414	23.7237	55.134	50
+	3415	21.1668	54.9295	48.0941
+	3416	13.1861	49.5029	46.5147
+	3420	12.2414	48.5569	46.935
+	3421	19.6743	54.5688	48.0938
+	3423	22.185	55.0795	50
+	3424	14.3496	50.8177	47.3873
+	3426	15.8399	52.2648	47.5644
+	3429	18.1471	53.9148	48.3149
+	3430	13.2878	49.73	47.9863
+	3432	16.9445	53.1582	48.0993
+	3435	20.6617	54.8622	50
+	3437	12.1727	48.584	48.4667
+	3440	19.187	54.4274	50
+	3442	17.8053	53.7537	50
+	3443	16.5279	52.8984	50
+	3444	13.2095	49.7053	50
+	3445	12.1501	48.5929	50
+	3446	14.2611	50.8259	50
+	3447	15.3515	51.9091	50
```

Comment:
Does this test have to be this huge?

Reply:
Is the same already existing test for quad-tri, but combined. And for the tri-quad I just inverted the model part order